.vscode/tasks.json

```diff
@@ -0,0 +1,147 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Install dependencies",
+      "type": "shell",
+      "command": "pnpm i",
+      "detail": "Install all dependencies using pnpm"
+    },
+    {
+      "label": "Build",
+      "type": "shell",
+      "command": "pnpm run build",
+      "detail": "Build all"
+    },
+    {
+      "label": "Custom build",
+      "type": "shell",
+      "command": "pnpm run build --filter=${input:workspaceName}",
+      "detail": "Build specific package or application"
+    },
+    {
+      "label": "Run tests",
+      "type": "shell",
+      "command": "pnpm run test",
+      "detail": "Run all tests without filter"
+    },
+    {
+      "label": "Run tests for a specific component",
+      "type": "shell",
+      "command": "pnpm run test ${input:componentName}",
+      "detail": "Run all tests for a specific component"
+    },
+    {
+      "label": "Open development servers",
+      "type": "shell",
+      "command": "pnpm dev",
+      "detail": "Open all development servers"
+    },
+    {
+      "label": "Run tests with filter",
+      "type": "shell",
+      "command": "pnpm run test --filter=${input:testFilter}",
+      "detail": "Run different test types"
+    }
+  ],
+  "inputs": [
+    {
+      "type": "pickString",
+      "id": "workspaceName",
+      "description": "Choose a worskspace",
+      "options": [
+        "css-workshop",
+        "react-workshop",
+        "portal",
+        "@itwin/itwinui-css",
+        "@itwin/itwinui-react",
+        "@itwin/itwinui-variables"
```

Comment:
We should include `vite-playground` and `e2e` in this list, because those are some of the most commonly used workspaces that we'd need to build for. And we can exclude `portal` as it's not important.

Reply:
I have made the change.